fastapi_sqla/base.py

```diff
@@ -20,6 +21,33 @@
 _ENGINE_KEYS_REGEX = re.compile(r"fastapi_sqla__(?!_)(.+)(?<!_)__(?!_).+")
 
 
+async def startup():
+    engine_keys = _get_engine_keys()
+    engines = {key: sqla.new_engine(key) for key in engine_keys}
+    for key, engine in engines.items():
+        if not _is_async_dialect(engine):
+            sqla.startup(key=key)
+        else:
+            await async_sqla.startup(key=key)
```

Comment:
Should we add tests for the async cases?

Reply:
hum, that's a lie

```
@mark.sqlalchemy("1.4")
@mark.require_asyncpg
async def test_startup_with_sync_and_async_sqlalchemy_url(
    async_session_key, sqla_startup_mock, async_sqla_startup_mock
):
    from fastapi_sqla.base import startup
    from fastapi_sqla.sqla import _DEFAULT_SESSION_KEY

    await startup()

    assert async_sqla_startup_mock.call_count == 1
    async_sqla_startup_mock.assert_awaited_once_with(key=async_session_key)

    assert sqla_startup_mock.call_count == 1
    sqla_startup_mock.assert_called_once_with(key=_DEFAULT_SESSION_KEY)
    ```